test/spec/features/stack/validate_spec.rb

```diff
@@ -3,9 +3,21 @@
     context 'entrypoint' do
       it 'sets stack entrypoint' do
         with_fixture_dir("stack/keywords") do
-          k = run 'kontena stack validate entrypoint.yml'
-          expect(k.code).to be_zero
-          expect(k.out).to match(/entrypoint.+?foo.+?entrypoint.+?foo2/m)
+          k = run! 'kontena stack validate --format=api-json entrypoint.yml'
+          data = JSON.load(k.out)
+
+          expect(data).to match hash_including(
+            'services' => a_collection_containing_exactly(
+              hash_including(
+                'name' => 'redis',
+                'entrypoint' => 'foo',
+              ),
+              hash_including(
+                'name' => 'redis2',
+                'entrypoint' => 'foo',
```

Comment:
Ugh, looks like I accidentially commited the test-to-make-sure-the-spec-actually-fails version, this should be `'entrypoint' => 'foo2'`